src/components/Datepicker.vue

```diff
@@ -340,9 +397,34 @@ export default {
      * Close the calendar
      */
     close() {
-      if (!this.isInline) {
-        this.view = ''
-        this.$emit('closed')
+      if (this.isInline) {
+        return
+      }
+
+      this.view = ''
+
+      if (this.showCalendarOnFocus || !this.focus.refs.length) {
+        this.$refs.dateInput.shouldToggleOnClick = true
+        document.body.focus()
```

Comment:
Yeah this seems better 👍

Reply:
Hmm, I submitted this change without first running my e2e tests...  Turns out the `ShowCalendarOnFocus #4-2` test fails after this change because the input field no longer has focus when you close a `show-calendar-on-focus` calendar by clicking on the input. I'll have a deeper look into this tomorrow...

Reply:
Ok - I've cleaned up the `close` method a little by using an `isClickOutside` flag instead of checking the length of the `focus.refs` array. This way, we only fire the `reviewFocus` method when `isClickOutside` is false. 

Integration tests pass now...